src/js/default-options.js

```diff
@@ -1,15 +1,12 @@
 /**
- * @file global-options.js
+ * @file default-options.js
```

Comment:
This file becomes the record of the initial video.js defaults. The global defaults currently active are always accessible at `videojs.options`.